mmv1/third_party/terraform/tests/resource_workstations_workstation_test.go.erb

```diff
@@ -0,0 +1,142 @@
+
+	context := map[string]interface{}{
+		"random_suffix": randString(t, 10),
+	}
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckWorkstationsWorkstationDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccWorkstationsWorkstation_basic(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+      {
+				Config: testAccWorkstationsWorkstation_modified(context),
+			},
+			{
+				ResourceName:            "google_workstations_workstation_cluster.default",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"etag"},
+			},
+		},
+	})
+}
+
+resource "google_workstations_workstation" "default" {
+  workstation_id         = "tf-test-workstation%{random_suffix}"
+  workstation_config_id  =  google_workstations_workstation_config.default.workstation_config_id
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location   		         = "us-central1"
+    display_name         = "workstation%{random_suffix}"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+`, context)
+}
+
+func testAccWorkstationsWorkstation_basic(context map[string]interface{}) string {
+	return Nprintf(`
+resource "google_compute_network" "default" {
+  name                    = "tf-test-workstation-cluster%{random_suffix}"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "tf-test-workstation-cluster%{random_suffix}"
+  ip_cidr_range = "10.0.0.0/24"
+  region        = "us-central1"
+  network       = google_compute_network.default.name
+}
+
+resource "google_workstations_workstation_cluster" "default" {
+  workstation_cluster_id     = "tf-test-workstation-cluster%{random_suffix}"
+  network                    = google_compute_network.default.id
+  subnetwork                 = google_compute_subnetwork.default.id
+  location   		             = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+
+resource "google_workstations_workstation_config" "default" {
+  workstation_config_id  = "tf-test-workstation-config%{random_suffix}"
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location   		         = "us-central1"
+
+  labels = {
+	  foo = "bar"
+  }
+}
+
+resource "google_workstations_workstation" "default" {
+  workstation_id         = "tf-test-workstation%{random_suffix}"
+  workstation_config_id  =  google_workstations_workstation_config.default.workstation_config_id
+  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
+  location   		         = "us-central1"
+  display_name           = "workstation-modified%{random_suffix}"
```

Comment:
Oh, the value here and in the other block should probably be flipped for clarity (we're starting from `workstation-modified` then removing `-modified` atm)

Reply:
Fixed.